index.html

```diff
@@ -1350,55 +1354,82 @@
           <p>
             The <dfn><code>state</code></dfn> attribute represents the
             <a>presentation connection</a>'s current state. It can take one of
-            the values of <a>PresentationConnectionState</a> depending on
-            connection state.
+            the values of <a>PresentationConnectionState</a> depending on the
+            connection state:
           </p>
+          <ul dfn-for="PresentationConnectionState">
+            <li>
+              <dfn>connecting</dfn> means that the user agent is attempting to
+              <a>establish a presentation connection</a> with the
+              <a>destination browsing context</a>. This is the initial state
+              when a <a>PresentationConnection</a> object is created.
+            </li>
+            <li>
+              <dfn>connected</dfn> means that the <a>presentation
+              connection</a> is established and communication is possible.
+            </li>
+            <li>
+              <dfn>closed</dfn> means that the <a>presentation connection</a>
+              has been closed, or could not be opened. It may be re-opened
+              through a call to <code><a for=
+              "PresentationRequest">reconnect</a>()</code>. No communication is
+              possible.
+            </li>
+            <li>
+              <dfn>terminated</dfn> means that the <a>presentation
```

Comment:
...that the receiving browsing context has been terminated.  The presentation connection is terminated and cannot be reopened.